helm-chart/amalthea/values.yaml

```diff
@@ -126,3 +126,14 @@ crdNames:
   singular: jupyterserver
   shortNames:
     - js
+
+
+# Determine how often amalthea checks each session to determine
+# whether it is idle and how long should one session be idle for
+# before it is culled. The total cpu usage of all containers in a 
+# user session's pod is used to determine if the session is idle
+# in addition to the open connections to the oauth2-proxy.
+culling:
```

Comment:
I think the default should be no culling. I can see many use-cases where this would be the desired behaviour. So either a `culling.enabled` flag or the option to omit culling by setting all numeric values to 0 would be nice.